project/Common.scala

```diff
@@ -39,7 +39,7 @@ object Common extends AutoPlugin {
       whitesourceGroup := Whitesource.Group.Community,
       crossVersion := CrossVersion.binary,
       crossScalaVersions := Dependencies.ScalaVersions,
-      scalaVersion := Dependencies.Scala212,
+      scalaVersion := Dependencies.ScalaVersions.head,
```

Comment:
This will obfuscate the actual Scala version used. The way `ScalaVersions` is built took me a little bit to figure out, but it seems in both cases when this is or isn't a cron build, the `Scala212` version will be used.

Reply:
Right, I reverted this.